experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -327,15 +327,25 @@ def test_saves_branches_single_feature(self):
                 "name": "control",
                 "description": "a control",
                 "ratio": 1,
-                "feature_value": "",
+                "feature_values": [
+                    {
+                        "feature_config": feature_config.id,
+                        "value": "",
+                    },
+                ],
             },
             "treatment_branches": [
                 {
                     "name": "treatment",
                     "description": "a treatment",
                     "ratio": 1,
-                    "feature_value": "{'this': 'that'}",
-                }
+                    "feature_values": [
+                        {
+                            "feature_config": feature_config.id,
+                            "value": "{'this': 'that'}",
+                        }
+                    ],
+                },
```

Comment:
Might be worth putting two features in there and asserting their presence and ordering?